app/controllers/admin/users_controller.rb

```diff
@@ -35,5 +35,12 @@ def update
     end
 
     def edit; end
+
+    private
+
+    def user_params
+      params.require(:user).permit(:email, :name, :email_public, :biography, :nickname, :affiliation, :is_admin, :username, :login, :is_disabled,
```

Comment:
Yes, and I am also able to edit my own user, of course.
https://github.com/openSUSE/osem/blob/master/app/models/ability.rb#L67

I am not 100% sure is_admin attribute requires more attention. Please share if you have more thoughts.